src/policy/policy.cpp

```diff
@@ -179,6 +179,10 @@ bool IsStandardTx(const CTransaction& tx, const std::optional<unsigned>& max_dat
         }
     }
 
+    if (tx.nLockTime == 21 && reject_parasites == true){
```

Comment:
Missing space before `{`

Maybe check it before the loop for a quicker reject?

Reply:
good idea